docs/rpc-ceph-install.md

```diff
@@ -0,0 +1,314 @@
+# Installation Procedures for Ceph in an RPC Customer Environment using rpc-ceph
+
+
+
+This process assumes that all servers already have either Ubuntu installed, all network configuration (incl bonding) is complete, and the servers are accessible via ssh.
+The servers involved in the deployment must be identified and the role(s) of each server clearly defined.  Every deployment needs "mons", "mgrs" and "osds".  Some deployments may also need RadosGW "rgws".  In a small deployment, a server may have all of these roles assigned.  In larger deployments, each server will typically be assigned fewer, or just a single role.
+In most cases, it is preferable to have separate networks for "Storage Front-end" (client access to the storage) and "Storage Back-end" (replication traffic within the cluster), but the "Storage Back-end" network is not required.
+The host from which the deployment will be managed must be identified.  This MAY be an "admin" server, "director" server or simply the first of the ceph storage servers, but it is USUALLY the first ceph storage server.  ALL of the following steps are to be performed on this deployment host.
+
+## Base Installation of Ceph using rpc-ceph
+
+### Create entries in /etc/hosts for all servers, specifying the IP address to which we can reach the servers via ssh:
+/etc/hosts
+Example **/etc/hosts** snippet:
+```bash
+192.168.0.101 ceph-stor01
+192.168.0.102 ceph-stor02
```

Comment:
We don't need these if we add ansible_ssh_host (see below) which i think we need anyway, as it stands right now.

Reply:
There may be other reasons to need this info populated in the /etc/hosts file.   This is exactly the kind of thing we need to review with the Deployment team

Reply:
Not fixed .  Pending discussion